cypress/e2e/DashboardDeleteLpa.feature

```diff
@@ -0,0 +1,53 @@
+@DashboardDeleteLpa
+Feature: DashboardDeleteLpa
+
+    I want to be able to delete an LPA on the dashboard
+
+    Background:
+        Given I ignore application exceptions
+
+    @focus
```

Comment:
What do we use `@focus` for?

Reply:
I didn't find any indication of where it's used or much in docs. I asked Nick yesterday and it seemed that anything not tagged with it wouldn't run, but I didn't tag my feature test with it and it did run in CI

Reply:
I think.